frontend/main.js

```diff
@@ -47,6 +47,7 @@ console.info('NODE_ENV:', process.env.NODE_ENV)
 
 Vue.config.errorHandler = function (err, vm, info) {
   console.error(`uncaught Vue error in ${info}:`, err)
+  if (process.env.NODE_ENV === 'development' || process.env.CI) throw err
```

Comment:
In development mode I don't think we need to rethrow as it will not accomplish anything but will cause duplicate messages in the console